src/seedu/addressbook/AddressBook.java

```diff
@@ -787,6 +835,29 @@ private static void addPersonToAddressBook(String[] person) {
         savePersonsToFile(getAllPersonsInAddressBook(), storageFilePath);
     }
 
+    /**
+     * Updates a person in the address book. Saves changes to storage file.
+     *
+     * @param person to update
+     */
+    private static boolean updatePersonInAddressBook(String[] person) {
+        Optional<String[]> targetPerson = Optional.empty();
```

Comment:
Nice to see usage of Optional (quite important in AB4) in your enhancement :) In this case however as this particular method is quite short, Optional may not be necessary but in longer code methods Optional will be especially useful.

Reply:
Okay thanks a lot!